docs/api/1.x/_details-patch-object.rst

```diff
@@ -10,10 +10,23 @@ changed meanwhile, a |status-412| error is returned.
     ``last_modified`` is updated to the current server timestamp, only if a
     field value was changed.
 
+
+Attributes merge
+----------------
+
+The provided values are merged with the existing object. For example:
+
+* ``{"a":"b"}`` + ``{"a":"c"}`` → ``{"a":"c"}``
+* ``{"a":"b"}`` + ``{"b":"c"}`` → ``{"a":"b", "b":"c"}``
+* ``{"a":"b"}`` + ``{"a":null}`` → ``{"a":null}`` : *attributes can't be removed with patch*
```

Comment:
Sorry forget this message, you are describing the current behavior and not the one we want :)